runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -15,3 +15,35 @@ pub mod testbed;
 // Prepares transactions and feeds them to the testbed in batches. Performs the warm up, takes care
 // of nonces.
 pub mod testbed_runners;
+
+use std::path::Path;
+
+use once_cell::sync::OnceCell;
+
+/// Lazily loads contract's code from a directory in the source tree.
+pub struct TestContract {
+    path: &'static str,
+    cell: OnceCell<Vec<u8>>,
+}
+
+impl TestContract {
+    pub const fn new(path: &'static str) -> TestContract {
+        TestContract { path, cell: OnceCell::new() }
+    }
+}
+
+impl std::ops::Deref for TestContract {
+    type Target = [u8];
+
+    fn deref(&self) -> &[u8] {
+        self.cell
+            .get_or_init(|| {
```

Comment:
Does this get called every time?  Or does `OnceCell` ensure that it's only loaded once?